packages/plugins/src/components/plugin-sidebar/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Button, Panel, Slot, Fill } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+//import PinnedPlugins from '../../header/pinned-plugins';
+import { withPluginContext } from '../plugin-context';
+import SidebarHeader from './sidebar-header';
+
+function PluginSidebarPinnedItems( { scope, ...props } ) {
+	return <Fill name={ `PluginPinnedAreas/${ scope }` } { ...props } />;
```

Comment:
It made me think that maybe we should support scopes natively in the SlotFill? What do you think? If I recall properly there are more places like that where we have to play with the name of the SlotFill pair.